src/components/button/button.scss

```diff
@@ -229,15 +229,16 @@
 }
 
 // button styles
-// solid
-:host([appearance="solid"]) {
+:host([appearance="solid"]),
+:host([appearance=""]) {
```

Comment:
Does this need the `appearance=""`? 

Would `[appearance]` be sufficient?

Reply:
If this is looking for when there is no appearance set, I think `:not([appearance])` would work. Or a css class could be used here.

Reply:
Looks like I’ve been testing for redundant cases, such as having the appearance set to an empty string. That’s not really an option and I’ve learned that we’ve decided not to address invalid cases. So I simplified the code quite a bit as a result.